packages/ai-jsx/src/batteries/sidekick/platform/system-message.tsx

```diff
@@ -18,28 +19,38 @@ export function SidekickSystemMessage({
   userProvidedGenUIUsageExamples,
   userProvidedGenUIComponentNames,
   includeNextStepsRecommendations,
+  hasKnowledgeBase,
   useCitationCard,
   outputFormat,
 }: SidekickSystemMessageProps) {
-  const daysOfWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
-  const currentDate = daysOfWeek[new Date().getDay()];
+  /* Format: 'Friday, Nov 10, 2023, 1:46:34 PM' */
+  const dateStringOptions = {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'short',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    hour12: true,
+    timeZone,
```

Comment:
If the user time zone were specified, will it be obvious to the model that this time is in the user's time zone? Wondering if this should be explicitly UTC or explicitly in the user's time zone.

Reply:
Hmm, I agree that the confusion and possibly different behaviour based on where the server is located is not good and I should fix that.

Still I'd rather the LLM doesn't do math here. Maybe I'll set it as PDT time if `timeZone` is not provided (to bias against our likely audience?).

Reply:
Done.